Cython/Utility/ImportExport.c

```diff
@@ -577,7 +577,12 @@ static int __Pyx_ImportFunction_$cyversion(PyObject *module, const char *funcnam
     d = PyObject_GetAttrString(module, "$api_name");
     if (!d)
         goto bad;
+#if PY_VERSION_HEX >= 0x030d0000
+    PyDict_GetItemStringRef(d, funcname, &cobj);
+#else
     cobj = PyDict_GetItemString(d, funcname);
+    Py_XINCREF(cobj);
+#endif
```

Comment:
Maybe a `__Pyx_PyDict_GetItemStringRef` is in order here as well? I'm not sure how much of this is okay/better to do.

Reply:
I think this stuff ends up in headers where it won't have access to the `__Pyx_...` utility code functions so it needs to be stand-alone (whatever the merits of defining `__Pyx_PyDict_GetItemStringRef` elsewhere).